jenkins/check-for-build.jenkinsfile

```diff
@@ -11,11 +11,10 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
-            H 1 * * * %INPUT_MANIFEST=1.3.5/opensearch-dashboards-1.3.5.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
-            H 1 * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
+            H/10 * * * * %INPUT_MANIFEST=2.2.0/opensearch-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch
+            H/10 * * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
             H 1 * * * %INPUT_MANIFEST=1.3.5/opensearch-1.3.5.yml;TARGET_JOB_NAME=distribution-build-opensearch
```

Comment:
Is 10 mins to high?, as the average build takes around 30 mins
https://build.ci.opensearch.org/job/distribution-build-opensearch/5856/
`H/60 * * * *` ---> 1hr should be good.

Reply:
H/10 is the default we used for a long time.
It will only trigger the check for build.
If distribution still running by previous trigger it will just halt.